tests/functional/pod_and_daemons/test_standalone_noobaa_pod_node_affinity.py

```diff
@@ -0,0 +1,59 @@
+import logging
+import pytest
+
+from ocs_ci.framework import config
+from ocs_ci.ocs import ocp, constants
+from ocs_ci.ocs.node import (
+    apply_node_affinity_for_noobaa_pod,
+)
+from ocs_ci.ocs.resources.pod import (
+    wait_for_pods_to_be_running,
+)
+from ocs_ci.framework.pytest_customization.marks import (
+    brown_squad,
+    tier1,
+    mcg_only_required,
+)
+
+log = logging.getLogger(__name__)
+
+
+@tier1
+@mcg_only_required
+@brown_squad
+class TestNoobaaPodNodeAffinity:
```

Comment:
IIRC, we have a separate job for noobaa standalone. Should this test be moved to https://github.com/red-hat-storage/noobaa-sa-ci/tree/main/tests? Please check with Red Squad

Reply:
@PrasadDesala Noobaa-SA-CI is for rpm based noobaa validation repo. This test is not suitable for that repo.

Reply:
Yes, We can move this test to mcg dir